parachain/runtime/tests/Cargo.toml

```diff
@@ -233,8 +233,8 @@ try-runtime = [
 	"snowbridge-system/try-runtime",
 	"sp-runtime/try-runtime",
 ]
-beacon-spec-mainnet = [
-	"snowbridge-ethereum-beacon-client/beacon-spec-mainnet",
+fast-runtime = [
```

Comment:
Lets use `beacon-spec-minimal` here so that the flag sounds like its to do with beacon chain as opposed to any of the runtimes.

Reply:
Agree!

Reply:
Then the `prod_or_fast` will not work
https://github.com/Snowfork/polkadot-sdk/pull/89/files#r1434875211

Btw, it's already used in Polkadot codebase everywhere(e.g. for a fast epoch time) and I would prefer to be in consistent with that.

Reply:
I don't see that problem. The BridgeHub runtime will continue to have a `fast-runtime` feature, that  will enable the `beacon-spec-mainnet` feature within the beacon client spec

Reply:
https://github.com/Snowfork/snowbridge/pull/1089/commits/d702ab8a72184fc5cb6fb433b169d0ced142487b